README.md

```diff
@@ -4,6 +4,7 @@
 [![Apache 2.0 licensed][apache-badge]][apache-url]
 [![Build Status][actions-badge]][actions-url]
 [![codecov](https://codecov.io/gh/xline-kv/xline/branch/master/graph/badge.svg)][cov-url]
+[![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7571/badge)](https://bestpractices.coreinfrastructure.org/projects/7571)
```

Comment:
please define the url seperately in [xx-badge] and [xx-url]

Reply:
Done